app/views/state_file/questions/nj_review/edit.html.erb

```diff
@@ -16,171 +16,195 @@
       </div>
-    </div>
+    </section>
   <% end %>
 
   <% if !current_intake.primary_veteran_unfilled? || !current_intake.spouse_veteran_unfilled? %>
-    <div id="veterans_exemption" class="white-group">
+    <section id="veterans_exemption" class="white-group">
       <div class="spacing-below-5">
-        <% if !current_intake.primary_veteran_unfilled? %>
-          <h2 class="text--body text--bold spacing-below-5"><%=t(".primary_veteran") %></h2>
-          <p><%= current_intake.primary_veteran %></p>
-        <% end %>
-        <% if !current_intake.spouse_veteran_unfilled? %>
-          <h2 class="text--body text--bold spacing-below-5"><%=t(".spouse_veteran") %></h2>
-          <p><%= current_intake.spouse_veteran  %></p>
-        <% end %>
+        <% unless current_intake.primary_veteran_unfilled? %>
+          <h2 class="text--body text--bold spacing-below-5"><%= t(".primary_veteran")  %></h2>
+          <p><%= current_intake.primary_veteran_yes? ? t("general.affirmative") : t("general.negative") %></p>
+        <%  end %>
+        <% unless current_intake.spouse_veteran_unfilled? %>
+          <h2 class="text--body text--bold spacing-below-5"><%= t(".spouse_veteran")  %></h2>
+          <p><%= current_intake.spouse_veteran_yes? ? t("general.affirmative") : t("general.negative") %></p>
+        <%  end %>
         <%= link_to StateFile::Questions::NjVeteransExemptionController.to_path_helper(return_to_review: "y"), class: "button--small" do %>
-          <%= t("general.edit") %>
+          <%= t(".review_and_edit") %>
           <span class="sr-only"><%= t(".edit_veteran") %></span>
         <% end %>
       </div>
-    </div>
+    </section>
   <% end %>
 
   <%# Dependents %>
 
-  <div id="medical_expenses" class="white-group">
+  <section id="medical_expenses" class="white-group">
     <div class="spacing-below-5">
       <h2 class="text--body text--bold spacing-below-5"><%=t(".medical_expenses") %></h2>
       <p><%= number_to_currency(current_intake.medical_expenses, precision: 0) %></p>
       <%= link_to StateFile::Questions::NjMedicalExpensesController.to_path_helper(return_to_review: "y"), class: "button--small" do %>
-        <%= t("general.edit") %>
+        <%= t(".review_and_edit") %>
         <span class="sr-only"><%= t(".medical_expenses") %></span>
       <% end %>
     </div>
-  </div>
+  </section>
 
-  <div id="property-tax" class="white-group">
+  <section id="property-tax" class="white-group">
     <div class="spacing-below-5">
       <h2 class="text--body text--bold spacing-below-5"><%=t(".property_tax_credit_deduction") %></h2>
 
-      <h3 class="text--body spacing-below-5"><%=t(".household_rent_own") %></h3>
-      <p><%= current_intake.household_rent_own %></p>
+      <h3 class="text--body spacing-below-5"><%=t(".household_rent_own", filing_year: current_tax_year) %></h3>
+      <p><%=t(".household_rent_own_#{current_intake.household_rent_own}") unless current_intake.household_rent_own_unfilled? %></p>
```

Comment:
[pebble] Should the unless be an if statement starting above the h3 instead? Do we want that header hanging out with an empty p tag if the field is unfilled?

Reply:
doesn't seem like a blocker, but that does seem like something to fix in the test, and not have unwanted code in the implementation for test case setup